src/main/java/it/unisa/Amigo/gruppo/controller/GruppoController.java

```diff
@@ -97,11 +138,127 @@ public String removeMembro(@PathVariable(name = "idPersona") int idPersona,@Path
         model.addAttribute("flagRimozione",1);
         model.addAttribute("personaRimossa",persona);
+        model.addAttribute("commissioni", gruppoService.findAllCommissioniByGruppo(idSupergruppo));
         return "gruppo/gruppo_detail";
     }
+
+    /***
+     * Rimuove un membro @{@link Persona} da una commissione @{@link Commissione}
+     * @param idPersona id della persona da rimuovere
+     * @param idSupergruppo id della commissione da cui rimuovere la persona
+     * @param model per salvare informazioni da recuperare nell'html
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/commissioni/{idSupergruppo}/remove/{idPersona}")
+    public String removeMembroCommissione(@PathVariable(name = "idPersona") int idPersona,@PathVariable(name = "idSupergruppo") int idSupergruppo, Model model){
+        Persona persona = gruppoService.findPersona(idPersona);
+        Supergruppo supergruppo = gruppoService.findSupergruppo(idSupergruppo);
+        gruppoService.removeMembro(persona,supergruppo);
+        Persona personaLoggata = gruppoService.getAuthenticatedUser();
+        prepareCandidateList(idSupergruppo,model,gruppoService.findAllMembriInSupergruppo(idSupergruppo));
+        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(),idSupergruppo));
+        model.addAttribute("isCapogruppo", gruppoService.isResponsabile(gruppoService.getAuthenticatedUser().getId(), gruppoService.findGruppoByCommissione(idSupergruppo).getId()));
+        model.addAttribute("flagRimozione",1);
+        model.addAttribute("personaRimossa",persona);
+        model.addAttribute("commissioni", gruppoService.findAllCommissioniByGruppo(idSupergruppo));
+        return "gruppo/commissione_detail";
+    }
+
+    /***
+     * Ritorna ad una pagina i membri @{@link Persona} di una commissione @{@link Commissione}
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idSupergruppo id della commissione di cui si vogliono visualizzare i membri
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/{id}/commissione_detail/{id_commissione}")
+    public String findAllMembriInCommissione(Model model, @PathVariable(name = "id")int idSupergruppo, @PathVariable(name = "id_commissione")int idCommissione){
+        Persona personaLoggata = gruppoService.getAuthenticatedUser();
+        prepareCandidateList(idCommissione,model,gruppoService.findAllMembriInSupergruppo(idCommissione));
+        model.addAttribute("isCapogruppo", gruppoService.isResponsabile(personaLoggata.getId(), gruppoService.findGruppoByCommissione(idCommissione).getId()));
+        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(),idCommissione));
+        return "gruppo/commissione_detail";
+    }
```

Comment:
Se gruppi e commissioni sono in una gerarchia, perché creare due metedi diversi per visualizare i dettagli e i membri del gruppo/commissione? Non sarebbe meglio creare un metodo polimorfico? (anche nel service)